decidim-core/app/views/layouts/decidim/user_profile.html.erb

```diff
@@ -11,15 +11,16 @@
           <div class="columns medium-4 large-3">
             <div class="side-panel">
               <ul class="tabs vertical side-panel__tabs" id="user-settings-tabs">
-                <%= user_profile_tab t("account", scope: "layouts.decidim.user_profile"), account_path, aria_link_type: :exact %>
-                <%= user_profile_tab t("notifications_settings", scope: "layouts.decidim.user_profile"), notifications_settings_path %>
+                <%= user_profile_tab t("account", scope: "layouts.decidim.user_profile"), decidim.account_path, aria_link_type: :exact %>
```

Comment:
We should put all this into the `user_menu` to make it more consistent with the rest of the code. That's the last change I ask for ;)

Reply:
Where should be the best place for defining this menus inside Decidim' source tree? For me creating new menus for a feature or a participatory space is clear, but this is outside of any of this scopes.